src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -52,9 +52,15 @@ function SubcellLimiterIDP(equations::AbstractEquations, basis;
                            positivity_variables_cons = [],
                            positivity_correction_factor = 0.1)
     positivity = (length(positivity_variables_cons) > 0)
-    number_bounds = length(positivity_variables_cons)
 
-    cache = create_cache(SubcellLimiterIDP, equations, basis, number_bounds)
+    bounds_order = Dict()
+    counter = 1
+    for index in positivity_variables_cons
+        bounds_order = Dict(bounds_order..., "$(index)_min" => counter)
```

Comment:
On a different note, why not just use
```suggestion
        bounds_order["$(index)_min"] = counter
```
instead of creating a new dict in each iteration? This seems much easier to understand to me...